### DISCUSSION TITLE
Version for ARM64?

### DISCUSSION THREAD

**Discussion by cssobral2013** [ikemen-engine/Ikemen-GO#909] [Feature Request]

**Comment by Windblade-GR01** [comment]
Currently the Engine supports ARM64 Linux.
The request is about providing pre-built Linux ARM64 builds on binary releases?

**Comment by cssobral2013** [comment]
Yes.

Em sex., 20 de mai. de 2022 às 15:55, Gacel ***@***.***>
escreveu:
